### PR TITLE
Add to documentation that HTML should not be used in instrument fields

### PR DESCRIPTION
**Pull request by mohini-tripathi**
## Brief summary of changes
- DQT - Rendering issue was faced in https://github.com/aces/Loris/issues/5092, Related clarification is documented to avoid it in the future.
#### Link(s) to related issue(s)
* Resolves #  
this fixes https://github.com/aces/Loris/issues/5355
#GSOC Aspirant
CC @christinerogers

### CODE REVIEW

File: modules/dataquery/README.md [hunk 1]
```diff
@@ -70,7 +70,8 @@ curl -H 'Content-Type: application/json' -X POST http://$YOURCOUCHDBADMIN:$YOURC
 
 ###### Import the LORIS instrument data
 
-This step is optional and not required if only the MRI portion of LORIS is used:
+This step is optional and not required if only the MRI portion of LORIS is used: \
```
Comment (PapillonMcGill) [inline modules/dataquery/README.md:73]: I would prefer to indicate when it is required. Other options might be invalidated in the future.
Similar change should also be applied to the MRI section below.
```suggestion
This step is optional and is required only if instruments are used: \
```

Comment (lingma) [reply modules/dataquery/README.md:73]: I believe the suggestion is not the original meaning, and contradictory.  @PapillonMcGill

Comment (christinerogers) [reply modules/dataquery/README.md:73]: @lingma you are right that it is not the original meaning, but @PapillonMcGill 's change is also more accurate and so better

Comment (christinerogers) [reply modules/dataquery/README.md:73]: however not relevant to this PR which I will close.

**Comment by christinerogers** [comment]
Thanks @mohini-tripathi for this suggestion -- we'd rather document where users look up how to create an instrument, so I am closing this PR.  
Thanks for your initiative here.

**Pull request closed without merging**